### PR TITLE
Feature/setup pip

### PR DESCRIPTION
**Pull request by smartwhatt**
**Features**
- Move Schema to this repo for reusability
- Allow pip insall via github

### CODE REVIEW

File: mysk_utils/schema/student_teacher/person.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+class EnglishPrefix(Enum):
+    """
+    English prefix for name
+    """
+
+    Mr = "Mr."
+    Mrs = "Mrs."
+    Miss = "Miss."
+    Master = "Master."
```
Comment (SiravitPhokeed) [inline mysk_utils/schema/student_teacher/person.py]: I don't think there's a period after master here.
(It looks like “Master.” appears multiple times, so if you wanna change this just be sure to change them all!)

File: mysk_utils/schema/student_teacher/schema.gql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    birthdate: Date!
+    citizen_id: String!
+    contacts: [Contact]!
+}
+
+type Teacher {
+    id: ID!
+    people: Person!
+    teacher_id: String!
+    subjectGroup: SubjectGroup!
```
Comment (SiravitPhokeed) [inline mysk_utils/schema/student_teacher/schema.gql]: Everything else uses snake_case but this uses camelCase.

File: mysk_utils/schema/subject/subjects.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+# I named this myself so idk if it is official
+class SubjectTypesEN(str, Enum):
+    """
+    SubjectTypes
+    """
+
+    core = "Core"
+    elective = "Elective"
+    additional = "Additional"
+    development = "Activity and Development"
```
Comment (SiravitPhokeed) [inline mysk_utils/schema/subject/subjects.py]: Went digging and found it’s actually “Learner’s Development Activities”.

**Review by SiravitPhokeed — APPROVED** [review]
Looks almost good to me!

**Review by Temp9699 — APPROVED** [review]
looking good

**Pull request merged**